src/Plugins/Reminder.php

```diff
@@ -10,28 +10,28 @@ class Reminder extends NuntiusPluginAbstract {
   protected $category = 'Reminders';
 
   public $formats = [
-    '/remind me next time I logging in to (.*)/' => [
+    '/remind me next time I log into (.*)/' => [
```

Comment:
It's supposed to be a conversation between two persons. Like Tony Stark talk with Jarvis(fun fact - Jarvis in the comics book is a real person, but in the movie, it's an AI).

I added him to the project.